### DISCUSSION TITLE
skywalking  start with no-init

### DISCUSSION THREAD

**Discussion by Evan-geng** [apache/skywalking#11524] [General]
hello
    when i use skywalking with 9.6.0,  After running for a while，it notice error message like “”[xxxx ](table: log does not exist. OAP is running in 'no-init' mode, waiting... retry 3s later)“. i find resolve in internet,But no optimal solution was found

**Comment by wu-sheng** [comment]
How are you using it? Don't expect upstream to have magic. If you don't debug for anything, we can't help.
